plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/AnnotationManager.java

```diff
@@ -265,16 +295,26 @@ public void onMapLongClick(@NonNull LatLng point) {
         }
       }
     }
+  }
 
-    @Nullable
-    private T queryMapForFeatures(@NonNull LatLng point) {
-      PointF screenLocation = mapboxMap.getProjection().toScreenLocation(point);
-      List<Feature> features = mapboxMap.queryRenderedFeatures(screenLocation, getAnnotationLayerId());
-      if (!features.isEmpty()) {
-        long id = features.get(0).getProperty(getAnnotationIdKey()).getAsLong();
-        return annotations.get(id);
-      }
-      return null;
+  @Nullable
+  private T queryMapForFeatures(@NonNull LatLng point) {
+    PointF screenLocation = mapboxMap.getProjection().toScreenLocation(point);
+    List<Feature> features = mapboxMap.queryRenderedFeatures(screenLocation, getAnnotationLayerId());
+    if (!features.isEmpty()) {
+      long id = features.get(0).getProperty(getAnnotationIdKey()).getAsLong();
+      return annotations.get(id);
+    }
+    return null;
+  }
+
+  @Nullable
+  T queryMapForFeatures(@NonNull PointF point) {
+    List<Feature> features = mapboxMap.queryRenderedFeatures(point, getAnnotationLayerId());
+    if (!features.isEmpty()) {
+      long id = features.get(0).getProperty(getAnnotationIdKey()).getAsLong();
+      return annotations.get(id);
     }
+    return null;
```

Comment:
there seems to be some shared code between these 2 methods.

Reply:
yup, going to fix this up